src/Expensify.js

```diff
@@ -163,7 +163,8 @@ class Expensify extends PureComponent {
         BootSplash
             .getVisibilityStatus()
             .then((status) => {
-                Log.info('[BootSplash] splash screen status', false, {status});
+                const appState = AppState.currentState;
+                Log.info('[BootSplash] splash screen status', false, {appState, status});
```

Comment:
Given that we use the return value of `getVisibilityStatus` for logic in the React level, would it not be preferable to return a bool, instead of a string?

Reply:
Currently `VisibilityStatus` is a union type `"visible" | "hidden" | "transitioning"`. I'd rather, even if slimming down all unused features here, align with the package current / future state, allowing you to migrate back to a future version later if you want.

Reply:
I guess that makes sense, thanks.